chapter14/qli/ex1.pl

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/perl
+use warnings;
+use strict;
+
+my @numbers = qw( 17 1000 04 1.50 3.14159 –10 1.5 4 2001 90210 666 );
+
+@numbers = @ARGV if ( @ARGV );
+
+printf ("%20s\n", $_) foreach ( sort {$a <=> $b} @numbers );
+# has warnings during sorting
+# Argument "M-bM-\0M-^S10" isn't numeric in sort at ./ex1.pl line 9.
```

Comment:
I had the same warning. I think if you copy the line from the book it introduces some control characters that you can't see.